docs/monthly-meeting/2024-03.md

```diff
@@ -0,0 +1,112 @@
+# Documentation Community Team Meeting (March 2024)
+
+- **Date:** 2024-03-05
+- **Time:** [20:00 UTC](https://arewemeetingyet.com/UTC/2024-03-05/20:00/Docs%20Meeting)
+- **This HackMD:** [https://hackmd.io/@encukou/pydocswg1](https://hackmd.io/@encukou/pydocswg1)
+- [**Discourse thread**](https://discuss.python.org/t/47109) (for March)
+- [**Meeting reports**](https://docs-community.readthedocs.io/en/latest/monthly-meeting/) (the latest one might be an [**unmerged PR**](https://github.com/python/docs-community/pulls))
+- **Calendar event:** (send your e-mail to Mariatta for an invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  To edit notes, click the “pencil” or “split view” button on the [HackMD document](https://hackmd.io/@encukou/pydocswg1). You need to log in (e.g. with a GitHub account).
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+
+## Roll call
+
+(Name / `@GitHubUsername` *[/ Discord, if different]*)
+
+- Ege Akman / `@egeakman`
+- Ashley Whetter / `@AWhetter`
+- Carol Willing / `@willingc`
+- Ned Batchelder / `@nedbat`
+- Hugo van Kemenade / `@hugovk`
+- Petr Viktorin / `@encukou`
+- Pradyun / `@pradyunsg`
+- Ezio Melotti / `@ezio-melotti`
+- Erlend Aasland / `@erlend-aasland`
+- CAM / `@CAM-Gerlach`
+
+## Reports and celebrations
+
+## Discussion
+
+  - Carol Willing: [Changelog splitting/reduction](https://github.com/python/docs-community/issues/98). At the February meeting, we had consensus that slimming down the changelog makes good sense. Let's discuss the potential approaches to do so, and try to reach consensus.
+  - CAM: split by minor release (feature version) would probably make most sense; we could make it more granular.
+  - Instead of changelog file we could have a changelog directory. Or we could chunk it in the directive. Either way is relatively straightforward.
+  - Ezio: We have to thing about the use of this changelog; one of those is searching for a particular module/function across all the releases. If we split it in several files, this'll be harder. Maybe we should keep a few latest releases on a single page. Or use tabs and make filtering faster.
+  - Hugo: On mobile, the page doesn't render well.
+  - CAM: Are people searching the rendered HTML, or other files. (Petr: i use Ctrl+F quite often)
+  - Carol: One more use case: When a bug happens, searching for when the feature was introduced.
+  - There are several issues: the time it takes to build, and the performance on mobile.
+  - Pradyun: We could make it so that there's still a search across changelogs. We could also add some Javascript to fetch the files and render them in one file.
```

Comment:
```suggestion
  - Pradyun: We could make it so that there's still a search across changelogs. We could also add some JavaScript to fetch the files and render them in one file.
```